File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/Substatus.java
package no.nav.personopplysninger.features.endreopplysninger.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Substatus {
    private String beskrivelse;
    private String domene;
    private String kode;
    private String referanse;
    private String status;

    public String getBeskrivelse() {
        return beskrivelse;
    }

    public String getDomene() {
        return domene;
    }

    public String getKode() {
        return kode;
    }

    public String getReferanse() {
        return referanse;
    }

    public String getStatus() {
        return status;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysninger/domain/Substatus.kt
package no.nav.personopplysninger.features.endreopplysninger.domain

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class Substatus {
    val beskrivelse: String? = null
    val domene: String? = null
    val kode: String? = null
    val referanse: String? = null
    val status: String? = null
}
